deployments/Dockerfiles/open_aea/start.sh

```diff
@@ -37,7 +35,9 @@ then
     echo "Installing the necessary dependencies!"
     aea install && cd .. && aea delete agent
 else
-    (aea add-key cosmos --connection && aea issue-certificates --aev) || (echo "Failed to add cosmos key needed for libp2p connection" && exit 1)
+    aea generate-key cosmos --connection
+    aea add-key cosmos --connection || (echo "Failed to generate the cosmos key needed for libp2p connection" && exit 1)
+    aea issue-certificates --aev || (echo "Failed to add cosmos key needed for libp2p connection" && exit 1)
```

Comment:
So this fixes the openssl legacy issue, we can create and add cosmos key but while issuing the certificates we get

```
    f"Cannot find private key with id '{ledger_id}'. Please use `aea generate-key {key_identifier}` and `aea add-key {key_identifier}` to add a private key with id '{key_identifier}'."
UnboundLocalError: local variable 'key_identifier' referenced before assignment
```

Reply:
Fixed it https://github.com/valory-xyz/open-autonomy/pull/1004/commits/1e2c167b42e4f0d19d9921d57789865709a752e5